### PR TITLE
chore(ut): BOOST_TEST -> BOOST_CHECK*s

### PR DESCRIPTION
**Pull request by iakovs**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,3 +17,6 @@ compile_commands.json
 *.pyc
 cmake-build-*/
 build/
+
+#vim
+.*.swp
```
[the remark this answers is not in the document]
Comment (iakovs) [reply .gitignore:22]: Done.

**Review by markaylett — CHANGES_REQUESTED** [review]
Please do not format external contributions in "contrib" directory because this complicates future diffs/review when pulling latest changes from upstream.

**Comment by iakovs** [comment]
> Please do not format external contributions in "contrib" directory because this complicates future diffs/review when pulling latest changes from upstream.

Sure thing. I didn't notice that. Then the first thing, I'll change the formatting script to exclude it

**Comment by iakovs** [comment]
Done

> Please do not format external contributions in "contrib" directory because this complicates future diffs/review when pulling latest changes from upstream.

Done. And clang-format changed to exclude `/contrib/`

**Review by markaylett — APPROVED** [review]

**Pull request merged**